feeadjuster/feeadjuster.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+import random
+import time
+from pyln.client import Plugin, Millisatoshi, RpcError
+
+
+plugin = Plugin()
+# Our amount and the total amount in each of our channel, indexed by scid
+plugin.adj_balances = {}
+# Cache to avoid loads of calls to getinfo
+plugin.our_node_id = None
+# Users can configure this
+plugin.update_treshold = 0.05
+
+
+def get_ratio(our_percentage):
+    """
+    Basic algorithm: the farther we are from the optimal case, the more we
+    bump/lower.
+    """
+    return 50**(0.5 - our_percentage)
+
+
+def maybe_adjust_fees(plugin: Plugin, scids: list):
+    for scid in scids:
+        our = plugin.adj_balances[scid]["our"]
```

Comment:
Must ensure that `scid in plugin.adj_balances`.

Reply:
Could be an assert as it's called just after maybe_populate_balances